src/styles/searchresults.css

```diff
@@ -0,0 +1,12 @@
+.search-results__images {
```

Comment:
Convention for CSS filenames is to name them in kebab case, just makes it easier to read